jvb/src/main/kotlin/org/jitsi/videobridge/websocket/config/WebsocketServiceConfig.kt

```diff
@@ -52,6 +52,15 @@ class WebsocketServiceConfig {
         }
     }
 
+    /**
+     * Whether compression (permessage-deflate) should be used for websockets
+     */
+    val useCompression: Boolean by config {
```

Comment:
I think if you use `@JvmField` here you'll get a nicer name when calling from java.

Reply:
`JvmField cannot be applied to delegated property`

Reply:
Ah, damn.  Usually when we are in this situation we make the field private and expose a better-named getter.  Like `fun shouldUseCompression() = useCompression`

Reply:
Changed to `fun shouldEnableCompression()` (rather than it being `getEnableCompression()` in Java).